ingest-app/src/cmr/ingest/services/granule_bulk_update/opendap/echo10.clj

```diff
@@ -44,12 +50,24 @@
     (when (seq urls)
       urls)))
 
+(defn- update-resource-type
+  "Transforms and returns the Type value, where the minor type is preserved."
+  [type-value]
+  (if type-value
+    (let [trim-with-default (fnil str/trim OPENDAP_RESOURCE_TYPE_MINOR)
+          minor (trim-with-default (second (str/split type-value #":")))]
+      (str OPENDAP_RESOURCE_TYPE_MAJOR " : " minor))
+    OPENDAP_RESOURCE_TYPE))
```

Comment:
I was unaware we had a use case for preserving subtype. As far as I can tell, this is a discrepancy between UMM-G and Echo10 (for update, append, and update-type)

Reply:
Checking with some folks on the appropriate behavior and will update accordingly.